.github/workflows/pr-checks.yml

```diff
@@ -26,16 +26,3 @@ jobs:
       - uses: ./.github/actions/install
 
       - uses: ./.github/actions/verify
-
-  check-commits:
```

Comment:
What was this check doing?

Reply:
This was verifying that the commits on the branch followed conventional commit styles with the label and description.

Unfortunetly, it wasn't really helpful because I want conventional commits on the main branch, not on the feature branches. So this wasn't checking that, and I set PRs to be sqaush only. So we'll just have to rely on people using conventional commits for the squash message.

Notice how this PR's title follows the style, so that when I go to squash and merge, the commit will already be in the correct format.

Reply:
Ah, gotcha. That makes a lot of sense.